src/html/contrast.html

```diff
@@ -1,12 +1,12 @@
 <p class="gpii-fd-instructions">You can choose a screen colour to make things easier to see.</p>
-<div class="gpii-fd-controls small-offset-2 medium-offset-3 small-7 medium-6 end text-center">
+<div class="gpii-fd-controls small-offset-2 medium-offset-1 small-7 medium-9 end text-center">
     <div class="gpii-fd-choice">
-        <input class="flc-prefsEditor-theme gpii-fd-hidden-accessible" name="theme" id="theme-default" value="default" type="radio" checked>
+        <input class="flc-prefsEditor-theme gpii-fd-hidden-accessible" name="theme" id="theme-default" value="default" type="radio">
         <span class="gpii-fd-icon gpii-fd-indicator"></span>
```

Comment:
On Nexus 7 running Android 5 in Chrome, the little arrow indicator appears above the selected option making it look broken.